apps/back-office/src/app/(dashboard)/share-documents/ReviewAndSend.tsx

```
@@ -0,0 +1,296 @@
+              >
+                View Message
+              </Button>
+            </DialogTrigger>
+            <DialogContent className="max-w-2xl">
+              <DialogHeader>
+                <DialogTitle>Email Preview</DialogTitle>
+              </DialogHeader>
+              <div className="mt-4 space-y-4">
+                <div className="prose prose-sm max-h-[60vh] overflow-y-auto">
+                  <div
+                    dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)